### DISCUSSION TITLE
Node Shutdown - 403 Forbidden: Permission check failed

### DISCUSSION THREAD

**Discussion by deltamelter** [dougiteixeira/proxmoxve#138] [Q&A] [answered]
Everything working great with this integration except when I made an automation to shutdown the whole node/server when the last VM stops running.  I just got this error  What permissions do I need to assign to my HA/proxmox user to allow this?
```
vm1 & Server Off : Error executing script. Unexpected error for device at pos 1: Proxmox node shutdown error - 403 Forbidden: Permission check failed (/nodes/daisy, Sys.PowerMgmt)
While executing automation automation.vm1_server_off
```

**Comment by deltamelter** [comment]
Added Sys.PowerMgmt to custom role an the action triggered, but only shutdown the node service, not the entire node like when I press shutdown node in ProxMox UI.  How do I do this?  I need the server to shutdown cleanly...

**Reply by dougiteixeira** [reply, marked as answer]
Some repairs have been added and will be available in the next version indicating the missing permissions for the user to be able to obtain data and carry out commands.

But in your specific case, the log is clear, you must include the `Sys.PowerMgmt` permission to the path `/nodes/daisy` for the user you are using in the integration. It is important to check that the user you are assigning permission to is actually the user being used by the integration, you can check this in the integration configuration options.
